docs/index.rst

```diff
@@ -15,5 +15,5 @@ Contents
    :maxdepth: 2
 
    topics/api/index
-   topics/install-olympia/index
+   topics/install/index
```

Comment:
You'll want to update the README's link to install docs to make sure the README doesn't break.